uber/templates/registration/new_base.html

```diff
@@ -147,7 +220,7 @@
         <td align="left">
             <a href="form?return_to=new%3f&id={{ attendee.id }}">{{ attendee.full_name }}</a>
             {% if attendee.banned %}
-                <div style="color:red">This attendee's name matches the name of a banned person.  Please get a department head to handle this.</div>
+                <div style="color:red">There is a problem with this registration. Please contact your department head.</div>
```

Comment:
Unrelated, but important -- we don't want to say someone is 'banned' when the watchlist is more flexible than that.